web/docs/auth/auth-hooks.md

```diff
@@ -30,6 +32,17 @@ If you are using OAuth, the flow includes extra steps before the signup flow:
   caption="OAuth Flow with Hooks"
 />
 
+<ImgWithCaption
+  source="/img/auth-hooks/login_flow_with_hooks.png"
+  alt="Login Flow with Hooks"
+  caption="Login Flow with Hooks"
+/>
+
+<small>
+
+\* When using the OAuth auth providers, the login hooks are both called before the session is created but the session is created quickly afterward, so it shouldn't make any difference in practice.
```

Comment:
This astrix isn't attached to anything.

Maybe put it on "Login Flow with Hooks*" (image caption).